samples/avro-samples/confluent-schema-registry-integration/confluent-schema-registry-integration-consumer/src/main/java/sample/consumer/ConfluentAvroConsumerApplication.java

```diff
@@ -0,0 +1,26 @@
+package sample.consumer;
+
+import java.util.function.Consumer;
+
+import com.example.Sensor;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class ConfluentAvroConsumerApplication {
+
+	private final Log logger = LogFactory.getLog(getClass());
```

Comment:
Maybe out of scope for this PR, but wdyt about using SL4J for our sample apps. In the past I have JCL for framework/lib code and SL4J for app code as its much nicer API.

Reply:
Spoke w/ @sobychacko offline and we will hold on "improvements + verify readme" for follow on PR(s) and use this PR strictly for "move the sample to new repo" functionality. I will create follow on ticket w/ details for this sample.

Reply:
Will be handled in #2376